docs/guides/connecting_to_your_data/cloud/gcs/spark.md

```diff
@@ -2,4 +2,153 @@
+
+</Prerequisites>
+
+## Steps
+
+### 1. Choose how to run the code in this guide
+
+<WhereToRunCode />
+
+### 2. Instantiate your project's DataContext
+
+Import these necessary packages and modules.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L3-L6
+```
+
+<SparkDataContextNote />
+
+Please proceed only after you have instantiated your `DataContext`.
+
+### 3. Configure your Datasource
+
+Using this example configuration, add in your GCS bucket and path to a directory that contains some of your data:
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'Python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L23-L42
+```
+
+Run this code to test your configuration.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L52
+```
+
+</TabItem>
+<TabItem value="python">
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_python_example.py#L21-L42
+```
+
+Run this code to test your configuration.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_python_example.py#L53
+```
+
+</TabItem>
+</Tabs>
+
+If you specified a GCS path containing CSV files you will see them listed as `Available data_asset_names` in the output of `test_yaml_config()`.
+
+Feel free to adjust your configuration and re-run `test_yaml_config()` as needed.
+
+### 4. Save the Datasource configuration to your DataContext
+
+Save the configuration into your `DataContext` by using the `add_datasource()` function.
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'Python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L54
+```
+
+</TabItem>
+<TabItem value="python">
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_python_example.py#L55
+```
+
+</TabItem>
+</Tabs>
+
+### 5. Test your new Datasource
+
+Verify your new Datasource by loading data from it into a `Validator` using a `BatchRequest`.
+
+<Tabs
+  defaultValue='runtime_batch_request'
+  values={[
+  {label: 'Specify a GCS path to single CSV', value:'runtime_batch_request'},
+  {label: 'Specify a data_asset_name', value:'batch_request'},
+  ]}>
+  <TabItem value="runtime_batch_request">
+
+Add the GCS path to your CSV in the `path` key under `runtime_parameters` in your `BatchRequest`.
```

Comment:
Just a tweak to be consistent with the class used in the example
```suggestion
Add the GCS path to your CSV in the `path` key under `runtime_parameters` in your `RuntimeBatchRequest`.
```

Reply:
Fixed (in next commit).  Thanks!